airflow/configuration.py

```diff
@@ -186,6 +186,18 @@ class AirflowConfigParser(ConfigParser):
         ('logging', 'worker_log_server_port'): ('celery', 'worker_log_server_port', '2.2.0'),
         ('api', 'access_control_allow_origins'): ('api', 'access_control_allow_origin', '2.2.0'),
         ('api', 'auth_backends'): ('api', 'auth_backend', '2.3'),
+        ('database', 'sql_alchemy_conn'): ('core', 'sql_alchemy_conn', '2.3'),
```

Comment:
Why do we want to add those values to "sensitive" list? I do not see it mentioned anywhere - neither updating nor decription of the PR. I also think it's not the best idea. 

Why this?

Reply:
We added these fields to `deprecated_options`.

Reply:
aaaaaaaaaaH. sorry. I have not noticed "folding" done by GitHub sorry :). It looked like it was added to sensitive_config_values. :D. My bad. disregard it please :)